xml/System.Diagnostics/Process.xml

```diff
@@ -703,23 +703,23 @@ process.BeginOutputReadLine();
       <Docs>
         <summary>Gets or sets whether the <see cref="E:System.Diagnostics.Process.Exited" /> event should be raised when the process terminates.</summary>
         <value>
-          <see langword="true" /> if the <see cref="E:System.Diagnostics.Process.Exited" /> event should be raised when the associated process is terminated (through either an exit or a call to <see cref="M:System.Diagnostics.Process.Kill" />); otherwise, <see langword="false" />. The default is <see langword="false" />.</value>
+          <see langword="true" /> if the <see cref="E:System.Diagnostics.Process.Exited" /> event should be raised when the associated process is terminated (through either an exit or a call to <see cref="M:System.Diagnostics.Process.Kill" />); otherwise, <see langword="false" />. The default is <see langword="false" />. Note that the <see cref="E:System.Diagnostics.Process.Exited" /> event might still be raised even if the value is <see langword="false" />.</value>
```

Comment:
@Priya91 I'm setting the text on this pass with "is raised" here, since I think you're saying that this is the *one* condition that will trigger the event regardless of the `EnableRaisingEvents` value. Let me know if it needs to go back to "might still be" if there are other conditions that may raise the event and this just happens to be one of them. Feel free to shoot back your own version of this.

> Note that the `Exited` event is raised even if the value of `EnableRaisingEvents` is `false` when the process exits during or before the user performs a `HasExited` check.